### PR TITLE
style-grid styling

### PR DESCRIPTION
**Pull request by ImrozNezami**
Different types of grid
Grid-2
Grid-3
Screenshot (229)

### CODE REVIEW

File: grid.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  .grid-2 {
+    grid-template-columns: 1fr 1fr;
+  }
+  .grid-3 {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+}
+.grid-content {
+  display: flex;
+  position: relative;
```
Comment (Sameony) [inline grid.css]: does this position relative have any particular reason to be here? i can't tell, so do look into it.

**Review by Sameony — COMMENTED** [review]
I was looking at the folder structure and there are some spelling mistakes, such as in stylesheets* folder and in the landingpage* documentation. Do correct them when possible

**Review by shuntzhere — APPROVED** [review]
Please check what Sameer has mentioned also try to implement base classes for recurring codes like flex and grid. That will help shorten your code, looks good anyways though.

File: grid.css (hunk 1 above)
Comment (ImrozNezami) [reply grid.css]: Removed

**Pull request merged**